src/init.cpp

```diff
@@ -149,6 +194,10 @@ hpx::program_options::options_description getOptionsDescription() {
                      "Number of normal priority CUDA streams per worker thread");
   desc.add_options()("dlaf:num-hp-cuda-streams-per-thread", hpx::program_options::value<std::size_t>(),
                      "Number of high priority CUDA streams per worker thread");
+  desc.add_options()("dlaf:mpi-mech",
+                     hpx::program_options::value<std::string>()->default_value("yielding"),
```

Comment:
Do you have benchmarks comparing yielding and polling now? I'd expect polling to never be worse than yielding.

Reply:
I had some old benchmarks that showed that they are approximately equivalent in the case of Cholesky (with perhaps a slight advantage for the polling approach). My intention was to have both for now until we can test more algorithms and then have a more clear picture if one approach is superior to the other.